src/tools/tidy/src/lib.rs

```diff
@@ -47,6 +47,7 @@ pub mod libcoretest;
 fn filter_dirs(path: &Path) -> bool {
     let skip = [
         "src/llvm",
```

Comment:
Probably can drop this line?

Reply:
I left the LLVM-related src/tools/* here too.  My thought was to keep ignoring those paths in tidy, in case a developer still has those obsolete submodules in place. (For instance, they may be moving between branches and don't want to remove them.)